snippets/URL-contains.js

```diff
@@ -0,0 +1,14 @@
+/* This JS snippet is for whether the URL contains specific text */
+
+
+/* if url contains the indicated text -> continue, otherwise throw an error */
+
+if(window.location.href.indexOf('<INSERT TEXT HERE>') < 0) {
```

Comment:
Let's turn "&lt;INSERT TEXT HERE&gt;" into a variable and then declare it at the top of the snippet.
Then it will be possible include that variable in the error message instead of `<INSERT YOUR TEXT>`